crates/admin/src/cluster_controller/service.rs

```diff
@@ -356,6 +365,48 @@ impl<T: TransportConnect> Service<T> {
         }
     }
 
+    fn is_active_controller(
```

Comment:
Maybe align `active` and `leader`. If we choose one term then there is only a single concept.